worldengine/common.py

```diff
@@ -120,3 +120,33 @@ def array_to_matrix(array, width, height):
         for x in range(width):
             matrix[y].append(array[y * width + x])
     return matrix
+
+def _equal(a, b):
```

Comment:
It is used in/as World.operator== - removing it would leave worlds without a proper way of checking for equality. I said this already at some point, but: I think this method should stay here.